galaxy/wrapper/test-data/profiler_result2.tabular

```diff
@@ -0,0 +1,3 @@
+bin labels		-0.0Kb	0.0Kb
```

Comment:
1. or 2. the only usage I have seen so far is in using the matrix not the other file.